src/index.html

```diff
@@ -2,16 +2,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="removeDef">
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        <div class="inputs">
+          <label>
+            Surename:
+            <input
+              type="text"
+              name="Surename"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="Name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            How old are you?
+            <input type="number"
+              name="Age"
+              max="100"
+              min="1"
+              value="12"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the aggrement
+            <input
+              type="checkbox"
+              name="acceptTermOfAggrement"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fields">
+        <legend>Registration</legend>
+        <div class="inputs">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="eMail: "
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Password:
+            <input
+              type="password"
+              name="Password"
+              required
+              minlength="5"
+              maxlength="32"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fields">
+        <legend>An interesting fact about you!</legend>
+        <div class="inputs">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label >
+            <input
+              type="radio"
+              name="loveCats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            What is your favorite color?
+            <input type="color" name="favouriteColor: ">
```

Comment:
keep names without special chars or spaces
```suggestion
            <input type="color" name="favouriteColor">
```